backend/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+using CbtBackend.Models;
+using CbtBackend.Entities;
+using CbtBackend.Services;
+
+namespace CbtBackend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
```

Comment:
We probably want this to be under `api/auth` no? Also, I wonder if there is a way to globally scope all endpoints to be under `api/`

Reply:
Actually no, this should be `/user/login` according to spec

Reply:
moved to `/user/login` in https://github.com/shilangyu/SE2-CBT/pull/124/commits/8792ee63e7775ea8690a607ffba87b44199356bf, we may want to make different routing rules later or merge this endpoint with another one, like a general `/user/*` endpoint